src/axes/timeAxis.ts

```diff
@@ -547,7 +548,8 @@ export module Axes {
         let leadingTickMark = visibleTickMarkRects[i];
         let trailingTickMark = visibleTickMarkRects[i + 1];
         if (!isInsideBBox(clientRect) || (lastLabelClientRect != null && Utils.DOM.clientRectsOverlap(clientRect, lastLabelClientRect))
-            || (leadingTickMark.right > clientRect.left || trailingTickMark.left < clientRect.right)) {
+            || (tierLabelPositions[i] === "between"
+            && (leadingTickMark.right > clientRect.left || trailingTickMark.left < clientRect.right))) {
```

Comment:
Instead of this fix, would it be better to check for collision using `leadingTickMark.top` / `leadingTickMark.bottom` / `trailingTickMark.top` / `trailingTickMark.bottom`?

Also if this expression gets too large, feel free to extract into a variable.